packages/forklift-console-plugin/src/modules/Providers/ProvidersPage.tsx

```diff
@@ -46,14 +46,14 @@ export const fieldsMetadata: Field[] = [
     sortable: true,
   },
   {
-    id: C.READY,
-    toLabel: (t) => t('Ready'),
+    id: C.PHASE,
+    toLabel: (t) => t('Status'),
     isVisible: true,
     filter: {
       type: 'enum',
       primary: true,
-      toPlaceholderLabel: (t) => t('Ready'),
-      values: fromI18nEnum(CONDITIONS),
+      toPlaceholderLabel: (t) => t('Status'),
+      values: fromI18nEnum(PROVIDER_STATUS),
```

Comment:
looks like order of `PROVIDER_STATUS`  list is lost ?

Reply:
`EnumFilter` is sorting by the labels. So this is a feature:)  
Details -> https://github.com/kubev2v/forklift-console-plugin/blob/9411bb0816eac004eb7092ba29f70b890e670653/packages/common/src/components/Filter/EnumFilter.tsx#L75